changelog.d/20240320_174524_andrew.yunxiang.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- Creating tasks with special characters in uploaded filename
```

Comment:
Please make sure that other upload-related functions work with such characters in file names.

Reply:
Bulk (non-TUS) uploads work fine. I'm not sure how to do remote and cloud uploads. Can I test those through the testing framework?

Reply:
Yes, it looks like the cypres test is the easiest way to initiate uploading via TUS.